### PR TITLE
Corrected pins and links for SK6812MINI-E schematic symbol

### PR DESCRIPTION
**Pull request by infused-kim**
Hi,

First of all, thank you so much for creating this library.

I think I may have found an issue with the SK6812MINI-E schematic symbol:
Screen Shot 2022-02-21 at 15 26 57

According to [the SK6812MINI-E datasheet](https://cdn-shop.adafruit.com/product-files/4960/4960_SK6812MINI-E_REV02_EN.pdf), the pins should be:
Screen Shot 2022-02-21 at 15 12 48

And the link in the symbol was also pointing to the MINI version instead of the MINI-E.

I have corrected both of these issues in this PR.

### CODE REVIEW

**Comment by ebastler** [comment]
Hi Kim, thanks for your input - this was, however, intentional. The footprint is used for both common anode and SK6812 Mini-E LEDs, and follows the pin numbering of the most widespread common anode LEDs, which is different from SK6812 pin order. The symbol has pin3/4 swapped to follow these changes.

The SK6812 symbol + footprint are tested and working the way I uploaded them. If I add your change to the symbol, I'd have to change the footprint as well so that it follows SK6812 Mini-E specs.

**Comment by infused-kim** [comment]
Thanks for the quick response.

To be honest I have difficulty understanding this. Perhaps it’s because I only started learning about electrical engineering about a week ago. 

If the pins of these symbols don’t match the pins of the actual SK6812MINI-E LED, then a board that intends to use those LED won’t work. 

Or am I misunderstanding something?

**Comment by ebastler** [comment]
The numbering in the symbol only says which pun number in the footprint is assigned to which symbol.
My footprint has pin 3 and 4 swapped when compared to the datasheet. So the pad where pin 4 of the LED will land is called pin3 on my footprint, and pin3 in my symbol. So basically I switched the pins in both the symbol and the footprint, compared the original. Swapping 3 and 4 two times results in the final pads being where they are supposed to be again.

So basically, I only called pin4 "3" and pin3 "4", but input/output/5V/GND are still routed to the correct pins in the end. I have a PCB with this symbol/footprint here - works.

**Comment by infused-kim** [comment]
Great, thank you so much for clarifying. That makes a lot of sense.

It may make sense to document this in the README as other people may be confused by it too.

**Comment by ebastler** [comment]
> It may make sense to document this in the README as other people may be confused by it too.

Yeah, the lib needs a bit of rework in general. I am currently very busy, will push a few changes when I find the time. One would be to introduce dedicated symbol libs for MX/Choc, with switch symbols with pre-assigned 1u footprints, and same for LED symbols with pre-assigned switch-led-footprints to make the workflow a bit faster.

Readme may need some additional explanations in general.

A rough explanation was added in the changelog for the update where I introduced the footprints, but I guess it is not really sufficient.
> Update note (11.12.2021): Dedicated SK6812 MINI-E footprints have been removed, and the SK6812 MINI-E symbol has been adapted to fit 6028R footprints. The pin numbering between 6028R and SK6812 MINI-E differs, so please take care to only use the updated symbol with these footprints, or your pinout will be wrong. The symbol included in marbastlib-mx will work, together with any of the 6028R footprints.
